RELEASE.txt

```diff
@@ -149,41 +103,17 @@ access), specifically to:
 
 - Push the new tag to GitHub::
 
-   git push upstream v<major>.<minor>.0
+   git push upstream v<major>.<minor>.<patch>
 
   (where ``upstream`` is the name of the
    ``github.com:scikit-image/scikit-image`` repository.)
 
-- Build the package wheels again, but change BUILD_COMMIT in both
-  Travis-CI and AppVeyor to the release version, e.g., ``v0.16.0``.
-
-- Upload the wheels to PyPI:
-
-  - Make sure ``twine`` is available. You can install it with
-    ``pip install twine``.
-  - Downlad ``wheel-uploader`` [1]_ and place it on your PATH.
-  - Make sure authentification with twine is working.
-    - See https://pypi.org/project/twine/
-    - Set your PyPI username in the environment variable TWINE_USERNAME
-    - Set your PyPI password in the environment variable TWINE_PASSWORD
-  - Run ``tools/upload_wheels.sh``.
-  - If everything worked, delete the wheels from your local directory with
-    ``rm -rf *.whl``
-
-.. [1] https://github.com/MacPython/terryfy/blob/master/wheel-uploader
-
-- Publish the source distribution on PyPi::
-
-   python setup.py sdist
-   twine upload dist/scikit-image-<major>.<minor>.0.tar.gz
+The previous step will
+- Upload the wheels to PyPI
 
-- Publish the release on github:
+- Publish the source distribution on PyPi
 
-  Go to https://github.com/scikit-image/scikit-image/releases and
-  click on "Draft a new release".
-  Choose the tag corresponding to this release, copy the release notes,
-  and upload the wheels in the appropriate slot. Finally, click on
-  "Publish release".
+- Publish the release on github
```

Comment:
As a future addition it will generate the release notes for inspection too, by using TownCrier